exonum-java-binding/common/src/main/java/com/exonum/binding/common/transaction/TransactionResult.java

```diff
@@ -32,8 +30,6 @@
  * if execution has failed.
  * Errors might be either service-defined or unexpected. Service-defined errors consist of an error
  * code and an optional description. Unexpected errors include only a description.
- *
- * @see TransactionExecutionException
```

Comment:
the link becomes broken because `TransactionExecutionException` locates in the core module.

Reply:
I see, would you add a back reference to TransactionExecutionException, if there isn't one already?

Reply:
I added a reference at `TransactionExecutionException` to this class